config/providers.php

```diff
@@ -4,21 +4,52 @@
 
 use App\Provider\CacheProvider;
 use App\Provider\EventDispatcherProvider;
+use App\Provider\FileRotatorProvider;
+use App\Provider\FileTargetProvider;
 use App\Provider\LoggerProvider;
 use App\Provider\MiddlewareProvider;
-use App\Provider\PSR17Provider;
+use App\Provider\Psr17Provider;
 use App\Provider\RouterProvider;
+use App\Provider\SessionProvider;
 use App\Provider\WebViewProvider;
 use Yiisoft\Arrays\Modifier\ReverseBlockMerge;
```

Comment:
`/* @var array $params */` would be handy here.

Reply:
done.